templates/partials/gmt-playwright-with-cache-v1.0.0.yml

```diff
@@ -0,0 +1,37 @@
+services:
+    gmt-playwright-nodejs:
+        image: mcr.microsoft.com/playwright:v1.56.0-noble
+#        volumes:
+#            - /tmp/.X11-unix:/tmp/.X11-unix # for debugging in non-headless mode
+        environment:
+            DISPLAY: ":0" # for debugging in non-headless mode
+        setup-commands:
+            # install playwright libraries in writeable directory
+            - command: cd /tmp/gmt-utils && npm init -y && npm install playwright@1.56.0
+              shell: bash
+        # playwright-ipc-v1.0.0.js will be mounted into container via lib/scenario_runner.py to not conflict with include directory restrictions
+
+    squid:
+        image: greencoding/squid_reverse_proxy:v5
+        healthcheck:
+            test: ["CMD", "curl", "-fs", "--proxy", "http://squid:3128", "https://www.google.com", "--insecure"]
+            interval: "1h" # effectively turns repeated healthchecks during runtime off
+            start_period: "60s"
+            start_interval: "1s"
+# activate for debugging
+#        ports:
+#           - 3128:3128
+
+flow-prepend:
+  - name: Starting browser IPC
+    container: gmt-playwright-nodejs
+    hidden: true
+    commands:
+      - type: console
+        command: node /tmp/gmt-utils/gmt-playwright-ipc-v1.0.0.js --headless true --browser firefox --proxy http://squid:3128
```

Comment:
Would be cool if there would be a way to also use chrome. But I can't think of anything easy and I wouldn't put more work into it. GMT Var defaults comes to mind.